.github/workflows/ci-build.yml

```diff
@@ -19,7 +19,7 @@ jobs:
          node-version: 'lts/iron'
          cache: 'yarn'
 
-      - run: yarn install --production=false --frozen-lockfile --ignore-scripts
+      - run: yarn install --immutable
```

Comment:
`--frozen-lockfile` is replaced by `--immutable`. This means the install will abort with an error exit code if the lockfile was to be modified by installing third party libraries.

Reply:
`--ignore-scripts` is replaced by `enableScripts: false` in .yarnrc.yml. This disables post-installation scripts from third party libraries.

Reply:
`--production=false` was originally used to ensure devDeps were not installed during ci-builds. However, it's not a flag that applies to Yarn Modern, and Yarn will not install any package listed in devDependencies if the `NODE_ENV` environment variable is set to `production`, which is the case in many of our build scripts anyway.

Reply:
Turns out that `--immutable` always installs all dependencies. An install can't be limited to production dependencies without changing the lockfile: https://github.com/yarnpkg/berry/issues/2253#issuecomment-748439061

`--production=false` should have ensured that devDeps were installed, though. Otherwise the subsequent build would fail, without the build tools.